File: app/src/main/java/space/pal/sig/old/repository/client/NasaClient.java
package space.pal.sig.old.repository.client;

import com.google.gson.Gson;

import javax.inject.Inject;

import okhttp3.OkHttpClient;

/**
 * SpaceNow
 * Created by Catalin on 7/19/2020
 **/
public class NasaClient extends BaseClient {

    private static final String NASA_URL = "https://api.nasa.gov";

    @Inject
    public NasaClient(OkHttpClient okHttpClient, Gson gson) {
        super(okHttpClient, gson);
    }

    @Override
    public String getApiUrl() {
        return NASA_URL;
    }
}

File: app/src/main/java/space/pal/sig/network/client/NasaClient.kt
package space.pal.sig.network.client

import space.pal.sig.BuildConfig

/**
SpaceNow
Created by Catalin on 12/6/2020
 **/
class NasaClient : BaseClient() {

    override fun getApiUrl(): String {
        return BuildConfig.NASA_API_URL
    }
}